instrumented/integration/src/main/kotlin/com/datadog/android/sdk/utils/IntentExtensions.kt

```diff
@@ -8,11 +8,18 @@ package com.datadog.android.sdk.utils
 
 import android.content.Intent
 import com.datadog.android.privacy.TrackingConsent
+import com.datadog.android.sessionreplay.SessionReplayPrivacy
 
 internal const val TRACKING_CONSENT_KEY = "tracking_consent"
+internal const val SR_PRIVACY_LEVEL = "sr_privacy_level"
+internal const val SR_SAMPLE_RATE = "sr_sample_rate"
 internal const val PENDING = 1
 internal const val GRANTED = 2
 internal const val NOT_GRANTED = 3
+internal const val ALLOW = 1
+internal const val MASK_USER_INPUT = 2
+internal const val MASK = 3
```

Comment:
are they needed? we can add `SessionReplayPrivacy` enum values to the intent extra directly and read them back using `getSerializableExtra`, because any enum is serializable.

Reply:
yes, check the tests as I have one for each of these values and pass those values in the IntentExtras :
https://github.com/DataDog/dd-sdk-android/pull/1593/files#diff-a691b85cf560e7461b22d88e7bd08c8d5ec192584d8da98cb159fb568c5e6ec4R28

Reply:
I am not sure we can pass an `Enum` in a Bundle, are we ?

Reply:
yes, we can, so these values are not needed.